DependencyInjection/Configuration.php

```diff
@@ -72,8 +71,15 @@ public function getConfigTreeBuilder()
                         ->scalarNode('stream_factory')->defaultNull()->end()
                     ->end()
                 ->end()
-            ->end()
-        ;
+                ->arrayNode('toolbar')
+                    ->canBeDisabled()
+                    ->addDefaultsIfNotSet()
+                    ->info('Extend the debug profiler with inforation about requests. This could only be active when the kernel is in debug mode.')
+                    ->children()
+                        ->scalarNode('formatter')->defaultNull()->end()
+                    ->end()
+                ->end()
+            ->end();
```

Comment:
same logic here, if we add another section, there will be an unnecessary diff on this line.